libs/dal/src/repositories/topic/topic.repository.ts

```diff
@@ -81,4 +81,24 @@ export class TopicRepository extends BaseRepository<EnforceEnvironmentQuery, Top
       _userId: userId,
     });
   }
+
+  async renameTopic(entity: Omit<TopicEntity, 'key'>): Promise<TopicEntity> {
```

Comment:
```suggestion
  async renameTopic(environmentId: string, topicId: string, name: string): Promise<TopicEntity> {
```

I actually think that this one is a bit clear on what parameters can go in, instead of passing the whole entity here. Also follows all our other dal methods. What do you think? `userId` is not needed here aswell since we might call this from the API key issued by multiple users.

Reply:
With the suggestion of change of the schema for Topic Subscribers I think it can make sense.
I would like us to write a document with the different standards for every layer so we can have consistency in the approaches.